CloneRepo.md

```diff
@@ -0,0 +1,37 @@
+# Setting up this repository in your local machine
+## Clonig a Repository
+
+When you create a repository on GitHub, it exists as a remote repository. You can clone your repository to create a local copy on your computer and sync between the two locations.
+Steps to follow are:
+
+1. On GitHub, navigate to the main page of the repository.
+
+2. Above the list of files, click  Code.
+
+![code](https://raw.githubusercontent.com/Tejas1510/Athavani/master/assets/images%20CloneRepo/code-button.png)
```

Comment:
We can use relative path here. For example: `assets/images%20CloneRepo/code-button.png`.